test/gallery.test.js

```diff
@@ -87,7 +73,7 @@ describe('Applying Gallery Filters - allow only 1 value per filter', () => {
   test('if user selected gender filters contains invalid value, it will not apply that filter', () => {
     let userSelectedFilters = { type: ['cat'], gender: ['extraterestru'] }
     expect(generateFilterOptions(userSelectedFilters)).toHaveProperty('gender', null)
-    userSelectedFilters = { type: ['cat'], gender: 'male' }
```

Comment:
Again. The input is invalid because it is not an arrest. Doesn't matter if the actual value is correct or not.
So this change is not required as well